docs/source/usage.rst

```diff
@@ -518,6 +518,36 @@ inputs we always return the same value. This behavior may be disabled by setting
     >>> assert c.uncached != c.uncached
 
 
+
+Custom interpolations can also receive the following special parameters:
+
+- ``_parent_`` : the parent node of an interpolation.
+- ``_root_``: The config root.
+
+This can be achieved by adding ``_parent_`` and ``_root_`` to the resolver signature.
+Note that special parameters must be defined as named keywords (after the `*`):
+
+
+.. doctest::
+
+    >>> OmegaConf.register_new_resolver(
+    ...     "sum_friends", 
```

Comment:
Minor: rename to `sum_siblings`? (more standard graph terminology)

Reply:
I guess we can just make this sum, where the feature over the previous sum is that this one treats non existing keys as 0.